tools/utility/dockerise/run.py

```diff
@@ -144,6 +144,8 @@ def _run(**kwargs):
             "--env",
             f"EDITOR={os.environ.get('EDITOR', 'nano')}",
             "--privileged",
+            "--group-add",
+            "dialout",
```

Comment:
```suggestion
            # Also need to create the group in the Dockerfile
            "--group-add",
            "dialout",
```

Reply:
Also, you can't do this, it makes any computer (i.e. mine) no longer able to build the code at all.

Reply:
@TrentHouliston  May I ask why? @Claegtun has been testing and building with this branch and it worked.

Reply:
I'm gonna guess he doesn't use a Mac

Reply:
He uses pop afaik, Im not really sure what I should do instead of this so it builds with anything. Is there a dialout equivalent for mac?

Reply:
> He uses pop afaik, Im not really sure what I should do instead of this so it builds with anything. Is there a dialout equivalent for mac?

I actually use Mint.

> @TrentHouliston May I ask why? @Claegtun has been testing and building in this branch with these lines and it worked.

Also, I don't think that I have been building from this branch as is. I had to merge with main to get it to build properly. I am not sure if that is to what @TrentHouliston is referring.

Reply:
How does this break it on Mac? Is it just that dialout is the wrong group?

Reply:
Dialout group doesn't exist so it just fails

Reply:
According to this you shouldn't need it on the host?
https://github.com/docker/cli/issues/4714#issuecomment-1881571231

Reply:
Have you rebuilt the image?

Reply:
Although there looks like there is a change to platform.py that is missing (but this missing should probably work in your favour)